pkg/template/servicebroker/bind.go

```diff
@@ -0,0 +1,144 @@
+package servicebroker
+
+import (
+	"errors"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/openshift/origin/pkg/openservicebroker"
+	templateapi "github.com/openshift/origin/pkg/template/api"
+	uservalidation "github.com/openshift/origin/pkg/user/api/validation"
+	kapi "k8s.io/kubernetes/pkg/api"
+	kerrors "k8s.io/kubernetes/pkg/api/errors"
+	kclientset "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+	"k8s.io/kubernetes/pkg/labels"
+	"k8s.io/kubernetes/pkg/selection"
+)
+
+// copied from vendor/k8s.io/kubernetes/pkg/kubelet/envvars/envvars.go
+func makeEnvVariableName(str string) string {
```

Comment:
I didn't/don't see an obvious utility method to use, and this is a placeholder implementation pending the appropriate return structures or conventions being defined in the openservicebroker API (@pmorie).  Long term I don't believe we should be returning environment-style strings here at all.